src/main/xerus/monstercat/api/Player.kt

```diff
@@ -114,7 +114,11 @@ object Player: FadingHBox(true, targetHeight = 25) {
 	fun reset() {
 		fadeOut()
 		if(!Files.isDirectory(Settings.PLAYEREXPORTFILE())) {
-			Files.write(Settings.PLAYEREXPORTFILE(), arrayListOf(""), StandardOpenOption.CREATE)
+			Files.newBufferedWriter(Settings.PLAYEREXPORTFILE(),
+					StandardOpenOption.TRUNCATE_EXISTING,
+					StandardOpenOption.WRITE,
+					StandardOpenOption.CREATE
+			).close()
 			logger.debug("Cleared export file (${Settings.PLAYEREXPORTFILE()}) from its contents")
```

Comment:
```suggestion
			logger.debug("Cleared export file (${Settings.PLAYEREXPORTFILE()})")
```
What else would you clear if not contents ^^

Reply:
Clearing a file could mean deleting it?

Reply:
Fixed in 9a4e5f5f81bf88c66e3fce8ea1b4ac5fd50152a2